indexx.js

```diff
@@ -1,6 +1,22 @@
-let entry = "0";
+let entry = "";
 let num1;
 let num2;
+let angle = "deg";
+let tmf = "o";
+let thf = "o";
+var fact = 1;
+let st_memory=0;
+
+function radians_to_degrees(radians)
+{
+  var pi = Math.PI;
+  return radians * (180/pi);
+}
+function degrees_to_radians(degrees)
+{
+  var pi = Math.PI;
```

Comment:
Pi is already available in `Math` class.
No need to define it explicitly.

Please format these function body brackets.